recommendations/index.md

```diff
@@ -135,6 +135,45 @@ Furthermore, multiple filters can be applied to a single request:
 GET /comments?filter[post]=1,2&filter[author]=12 HTTP/1.1
 ```
 
+## <a href="#including-links" id="including-links" class="headerlink"></a> Including Top-level, Resource-level and Relationship Links
+
+The base specification is agnostic about including links with a resource response. However it is recommended that `links` the following links are always included within the response for a resource.
```

Comment:
Please change this last sentence to something like the following and add a colon at the end, like:

> However, it is recommended that the following links be included within response documents: